src/components/Button.js

```diff
@@ -2,21 +2,22 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import './Calculator.css';
 
-const Button = ({ label }) => {
```

Comment:
![image](https://github.com/CVILLA09/Math-Magicians/assets/93039359/a22a034c-98ea-4465-931c-cc9a72dec1c8)

- [ ] It would be nice if you fix the FAQ link from your readme file to make your readme descriptive and professional 👍 If you don't have any dedicated section then kindly remove this FAQ link from your `Table of Content` 😊